tools/c2chapel/c2chapel.py

```diff
@@ -374,7 +386,10 @@ def genStructOrUnion(structOrUnion, name=""):
         if innerStructOrUnion is not None:
             genStructOrUnion(innerStructOrUnion)
 
-        fieldName = getDeclName(decl)
+        if type(decl) is not c_ast.Pragma:
+            fieldName = getDeclName(decl)
+        else:
+            break
```

Comment:
Might be a little cleaner to reverse this so that it's `if pragma break` and leave the fieldName code at it's original indentation